services/src/error.rs

```diff
@@ -521,3 +524,9 @@ impl From<tokio::task::JoinError> for Error {
         Error::TokioJoin { source }
     }
 }
+
+impl From<opentelemetry::trace::TraceError> for Error {
+    fn from(source: opentelemetry::trace::TraceError) -> Self {
+        Error::Trace { source }
+    }
+}
```

Comment:
Use `#[snafu(context(false))]` instead

Reply:
why? maybe we want to see why the tracing failed.

Reply:
But you are implementing exactly what context(false) auto-implements

Reply:
ed702fe821950f56ee1a6ab0d56aacc25adbf96c